### ISSUE TITLE
background

### ISSUE THREAD

**Issue by julien9999** [adampax/TrackLocation#2]
Hi, will it work if the app is fully closed ?
thanks

**Comment by adampax** [comment]
Yes, even when app is closed and fully removed from background, or when device is restarted.

[Significant-change location services](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/LocationAwarenessPG/CoreLocation/CoreLocation.html#//apple_ref/doc/uid/TP40009497-CH2-SW8)

**Comment by julien9999** [comment]
Perfect. Thanks. 
What would be the equivalent for android ?

**Comment by adampax** [comment]
On Android, I've use the bencoding.AlarmManager module to run a service at regular intervals that will start GPS, get location, and turn itself off. 

The AlarmManager will continue to run the service even when the app is removed from background, but not when the device is restarted.  You can monitor device restarts with another module, bencoding.android.tools, that will allow you to register a BootReceiver that will restart the service above when the device restarts.

Android is more complicated to set up, but the tradeoff is you do get location updates at regular intervals, not just on a significant location change.

**Comment by julien9999** [comment]
Perfect thanks I'll take a look into it. 
Unless you have done code to share too ?

**Comment by adampax** [comment]
Sorry, don't have any handy
